index.js

```diff
@@ -0,0 +1,35 @@
+var koa = require('koa');
+var Github = require('github-api');
+var app = koa();
+
+var github = new Github({
```

Comment:
@danmactough https://help.github.com/articles/creating-an-access-token-for-command-line-use/